templates/csharp/message-extension-copilot/appPackage/manifest.json.tpl

```diff
@@ -37,12 +37,14 @@
                     "description": "Find NuGet package according to the NuGet package name",
                     "title": "Find NuGet Package",
                     "type": "query",
+                    "semanticDescription": "This command retrieves detailed information about an npm package using the provided npm package name.",
```

Comment:
npm package? Since the description is conflict with the command title `Find NuGet Package`.

Reply:
Updated. Thanks for pointing out.